src/shell/hooks/useFilePath.js

```diff
@@ -0,0 +1,47 @@
+import zuid from "zuid";
+import { useSelector } from "react-redux";
+
+export function useFilePath(ZUID = "") {
+  const getFiles = state => state.files;
+  const files = useSelector(getFiles);
+  if (!ZUID) {
+    throw new Error("Required ZUID argument was not provided");
+  }
+
+  switch (Number(ZUID.split("-")[0])) {
+    case zuid.prefix.SITE_CONTENT_SET:
+      let findfile = files.find(file => file.contentModelZUID === ZUID);
+      if (findfile) {
+        return `/code/file/views/${findfile.ZUID}`;
+      } else {
+        return "/code";
+      }
+
+    case zuid.prefix.SITE_CONTENT_ITEM:
+      let findFile = files.find(file => file.modelZUID === ZUID);
```

Comment:
This logic is invalid. It would compare a model zuid (6-) against a provided item zuid (7-)

Reply:
Removed SITE_CONTENT_ITEM since all use cases for edit code link use SITE_RESOURCE to check relative file path